v1/nix/modules/drvs/apache-airflow/default.nix

```diff
@@ -6,7 +6,7 @@
   l = lib // builtins;
   python = config.deps.python;
   extractPythonAttrs = config.nixpkgs-overrides.lib.extractPythonAttrs;
-  nixpkgsAttrs = extractPythonAttrs python.pkgs.apache-airflow;
+  nixpkgsAttrs = extractPythonAttrs config.deps.apache-airflow;
```

Comment:
This conflicts with my recent changes to this package in https://github.com/nix-community/dream2nix/pull/529

Reply:
I'm not quite sure why. The package seems to build fine

Reply:
You don't need it anymore, when building apache-airflow with the new python mod

Reply:
removed it